Packs/CortexXDR/TestPlaybooks/Test_Playbook_-_Cortex_XDR_Malware_-_Incident_Enrichment.yml

```diff
@@ -0,0 +1,4565 @@
+    isautoswitchedtoquietmode: false
+  "180":
+    id: "180"
+    taskid: 3f7806ed-77f7-4aa6-8c3a-2800336f36b0
+    type: condition
+    task:
+      id: 3f7806ed-77f7-4aa6-8c3a-2800336f36b0
+      version: -1
+      name: Verify Device OS Name
+      description: |
+        Verify that the ‘deviceosname’ incident field was filled out correctly.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "184"
+      Verified:
+      - "136"
+    separatecontext: false
+    conditions:
+    - label: Verified
+      condition:
+      - - operator: isEqualString
+          left:
+            value:
+              simple: incident.deviceosname
+            iscontext: true
+          right:
+            value:
+              complex:
+                root: Endpoint
+                accessor: OS
+            iscontext: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 5760,
+          "y": -380
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "182":
+    id: "182"
+    taskid: 3511a1b2-fd06-4375-87a4-af6be9ca684d
+    type: condition
+    task:
+      id: 3511a1b2-fd06-4375-87a4-af6be9ca684d
+      version: -1
+      name: Verify Device Status
+      description: |
+        Verify that the ‘devicestatus’ incident field was filled out correctly.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "189"
+      Verified:
+      - "136"
+    separatecontext: false
+    conditions:
+    - label: Verified
+      condition:
+      - - operator: isEqualString
+          left:
+            value:
+              simple: incident.devicestatus
+            iscontext: true
+          right:
+            value:
+              complex:
+                root: Endpoint
+                accessor: Status
+            iscontext: true
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 8190,
+          "y": -380
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "183":
+    id: "183"
+    taskid: c648bd59-a10f-4cff-8e8d-422821ed1b39
+    type: condition
+    task:
+      id: c648bd59-a10f-4cff-8e8d-422821ed1b39
+      version: -1
+      name: Verify Isolated
+      description: |
+        Verify that the ‘isolated’ incident field was filled out correctly.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "190"
+      Verified:
+      - "136"
+    separatecontext: false
+    conditions:
+    - label: Verified
+      condition:
+      - - operator: isEqualString
+          left:
+            value:
+              simple: incident.isolated
```

Comment:
I'm not sure we are setting this in the incident enrichment PB. Can you check pls that we are indeed setting this up through the PB?